// TODO(hjd): Autogenerate.

syntax = "proto3";

import "print.proto";
import "sched_switch.proto";

message FtraceEvent {
  uint32 pid = 1;
  oneof event {
    PrintFtraceEvent print = 2;
    SchedSwitchFtraceEvent sched_switch = 3;
  }
}
